run-container.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+dir=$(dirname $(readlink -f ${BASH_SOURCE[0]}))
+
+docker run --privileged \
```

Comment:
@arturkow2000 the privileged flag is given twice - is it intended?

Reply:
Oversight, fixed.